build.gradle

```diff
@@ -1,70 +1,69 @@
 plugins {
-	id 'fabric-loom' version '1.4-SNAPSHOT'
-	id 'maven-publish'
+	id "architectury-plugin" version "3.4-SNAPSHOT"
+	id "dev.architectury.loom" version "1.4-SNAPSHOT" apply false
 }
 
-base.archivesName = project.archives_base_name
-version = project.mod_version
-group = project.maven_group
-if ("false".equalsIgnoreCase(System.getenv("IS_RELEASE"))) {
-	version += "-SNAPSHOT"
+architectury {
+	minecraft = rootProject.minecraft_version
 }
 
-dependencies {
-	minecraft "com.mojang:minecraft:${project.minecraft_version}"
-	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
-	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
-}
-
-processResources {
-	inputs.property "version", version
+subprojects {
+	apply plugin: "dev.architectury.loom"
 
-	filesMatching("fabric.mod.json") {
-		expand "version": version
+	dependencies {
+		minecraft "com.mojang:minecraft:${rootProject.minecraft_version}"
+		mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
 	}
-}
 
-// ensure that the encoding is set to UTF-8, no matter what the system default is
-// this fixes some edge cases with special characters not displaying correctly
-// see http://yodaconditions.net/blog/fix-for-java-file-encoding-problems-with-gradle.html
-tasks.withType(JavaCompile).configureEach {
-	options.encoding = "UTF-8"
+	// TODO: jar { from("LICENSE") }?
```

Comment:
Try to keep the license file inside the jar

Reply:
done